portfolio/src/main/webapp/index.html

```diff
@@ -2,16 +2,27 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Carmen's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Carmen's Portfolio</h1>
+      <a href="https://www.linkedin.com/in/carmen-benitez-0b409a171">LinkedIn Profile</a>
+      <a href="https://github.com/carmenBen">GitHub Profile</a>
+      <p>
+          Hi my name is Carmen! I am a 20 year old  living in New York City.
+          I was born in the Philippines, but moved to Australia when I was four! 
+          I lived in Sydney until I was 14, and then I moved to New York.
+          I am a rising junior at Harvey Mudd College where I am a joint majoring
+          in Computer Science and Math. I am concentrating(like a minor) in
+          economics too. In my free time I enjoy reading, listening to music,
+          and watching movies! I also love going to concerts. 
+      </p>
+      <img src="images/selfie.JPG" width="200" height="200" />
+      <p>Click here to get a random album suggestion:</p>
```

Comment:
Since you changed this to an album suggestion, I would also change all the function names, variable names and element ids to reflect what you're actually going to be displaying.